### PR TITLE
[JENKINS-58290] Verifying ContainerExecDecorator scalability

### PR DESCRIPTION
**Pull request by jglick**
[JENKINS-58290](https://issues.jenkins-ci.org/browse/JENKINS-58290)

Integration test for https://github.com/jenkinsci/durable-task-plugin/pull/98.

### CODE REVIEW

**Comment by jglick** [comment]
https://github.com/jenkinsci/durable-task-plugin/pull/97 seems to fix it, though a rewrite of `ContainerExecDecorator` might be better.

**Comment by jglick** [comment]
Interactively, run

```sh
while :; do jstack $pid | fgrep '#' | sort | egrep -i ok\|pool > /tmp/x; clear; cat /tmp/x; sleep 5; done
```

on

```groovy
while (true) {
    podTemplate(label: BUILD_TAG, containers: [containerTemplate(name: 'ubuntu', image: 'ubuntu', command: 'sleep', args: 'infinity')]) {
        node (BUILD_TAG) {
            container('ubuntu') {
                branches = [:] // TODO cannot use collectEntries because: java.io.NotSerializableException: groovy.lang.IntRange
                for (int x = 0; x < 1000; x += 5) {
                    def _x = x
                    branches["sleep$x"] = {
                        sleep time: _x, unit: 'SECONDS'
                        sh """set +x; while :; do echo -n "$_x "; date; sleep 10; done"""
                    }
                }
                parallel branches
            }
        }
    }
}
```

**Comment by Vlatombe** [comment]
Pending a release of durable-task 1.30

File: pom.xml [hunk 1]
```diff
@@ -94,10 +95,10 @@
       <version>6.8</version>
     </dependency>
 
-    <dependency> <!-- OnceRetentionStrategy -->
+    <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>durable-task</artifactId>
-      <version>1.29</version>
+      <version>1.30-rc421.8a00737f12b2</version> <!-- TODO https://github.com/jenkinsci/durable-task-plugin/pull/98 -->
```
Comment (dwnusbaum) [inline pom.xml]:
```suggestion
      <version>1.30</version>
```

**Pull request merged**